LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2017 Sixhours
+   Copyright {yyyy} {name of copyright owner}
```

Comment:
I think you should change this line to: `Copyright [yyyy] [name of copyright owner]`